uds/transport_interface/can_transport_interface.py

```diff
@@ -466,7 +474,7 @@ def n_bs_measured(self) -> Optional[TimeMillisecondsAlias]:
         return self.__n_bs_measured
 
     @property  # noqa
-    def n_cr_measured(self) -> Optional[TimeMillisecondsAlias]:
+    def n_cr_measured(self) -> Optional[Tuple[TimeMillisecondsAlias, ...]]:
         """
         Get the last measured value of N_Cr time parameter.
```

Comment:
add reference to knowledge base N_Cr time explanation (same for other time parameters)